src/lxc/lxccontainer.c

```diff
@@ -583,6 +583,9 @@ static int do_lxcapi_console_log(struct lxc_container *c, struct lxc_console_log
 {
 	int ret;
 
+	if (!c)
+		return -1;
```

Comment:
Thanks for the PR.
Seeing how the rest of this function is returning negative errno codes this should probably return `-EINVAL`.

Reply:
@Blub It's better. Thanks for your review.:)

Reply:
Good point. Set errno or negative return in API calls?